packages/babel-parser/test/fixtures/experimental/module-blocks/invalid-module-bracketBarL/input.js

```diff
@@ -0,0 +1 @@
+module{|foo:42|}}
```

Comment:
Before this PR we parses this input as if it were
```js
// note the extra required `{` after `module`
module {
  {| foo: 42 |}
}
```
This is now fixed by expecting `{` after `module`.